charts/traccar/templates/configmap.yaml

```diff
@@ -13,6 +13,7 @@ data:
     <!DOCTYPE properties SYSTEM 'http://java.sun.com/dtd/properties.dtd'>
     <properties>
         <entry key='config.default'>./conf/default.xml</entry>
+        <entry key='config.useEnvironmentVariables'>true</entry>
```

Comment:
@tananaev by adding this configuration in the xml file, is it possible to use environment variables?

Reply:
Yes.